exporter/splunkhecexporter/factory.go

```diff
@@ -71,9 +86,13 @@ func createDefaultConfig() component.Config {
 			HTTP2ReadIdleTimeout: defaultHTTP2ReadIdleTimeout,
 			HTTP2PingTimeout:     defaultHTTP2PingTimeout,
 		},
-		SplunkAppName:           defaultSplunkAppName,
-		BackOffConfig:           configretry.NewDefaultBackOffConfig(),
-		QueueSettings:           exporterhelper.NewDefaultQueueSettings(),
+		SplunkAppName: defaultSplunkAppName,
+		BackOffConfig: configretry.NewDefaultBackOffConfig(),
+		QueueSettings: exporterhelper.NewDefaultQueueSettings(),
+		BatcherConfig: exporterbatcher.Config{
+			Enabled:      false,
+			FlushTimeout: 200 * time.Millisecond, // need default value to prevent validation failures
```

Comment:
Let's use `exporterbatcher.NewDefaultConfig()`, but disabled
```
batcherCfg := exporterbatcher.NewDefaultConfig()
batcherCfg.Enabled = false
```

Reply:
Because `MinSizeConfig` has to be provided, otherwise it's noop

Reply:
Fixed the initialization.